lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -2,6 +2,8 @@
 # frozen_string_literal: true
 
 require "pathname"
+Tapioca.silence_output { require "pry-doc" }
```

Comment:
This is necessary because `pry-doc` prints directly to `STDOUT` if it doesn't find the exact folder of documentation for the current Ruby version.